pkgs/development/libraries/libdeflate/default.nix

```diff
@@ -23,11 +36,12 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  passthru.tests.static = pkgsStatic.libdeflate;
   meta = with lib; {
     description = "Fast DEFLATE/zlib/gzip compressor and decompressor";
     license = licenses.mit;
     homepage = "https://github.com/ebiggers/libdeflate";
     platforms = platforms.unix;
-    maintainers = with maintainers; [ orivej ];
+    maintainers = with maintainers; [ orivej kaction ];
   };
-}
+} // (if stdenv.hostPlatform.isStatic then staticArgs else { }))
```

Comment:
Can we just throw this onto staging to avoid this hackery?

Reply:
Done.